File: library/src/main/java/me/proxer/library/enums/License.java
package me.proxer.library.enums;

import com.squareup.moshi.Json;

/**
 * Enum holding the available license types.
 *
 * @author Ruben Gees
 */
public enum License {
    @Json(name = "0")UNKNOWN,
    @Json(name = "1")NOT_LICENSED,
    @Json(name = "2")LICENSED
}

File: library/src/main/kotlin/me/proxer/library/enums/License.kt
package me.proxer.library.enums

import com.serjltt.moshi.adapters.FallbackEnum
import com.squareup.moshi.Json

/**
 * Enum holding the available license types.
 *
 * @author Ruben Gees
 */
@FallbackEnum(name = "UNKNOWN")
enum class License {

    @Json(name = "0")
    UNKNOWN,

    @Json(name = "1")
    NOT_LICENSED,

    @Json(name = "2")
    LICENSED
}
